oas_docs/output/kibana.serverless.yaml

```diff
@@ -82290,6 +82262,36 @@ components:
       required:
         - command
         - config
+    Security_Detections_API_QueryAlertsBodyParams:
+      type: object
+      properties:
+        _source:
+          oneOf:
+            - type: boolean
+            - type: string
+            - items:
+                type: string
+              type: array
+        aggs:
+          additionalProperties: true
+          type: object
+        fields:
+          items:
+            type: string
+          type: array
+        query:
+          additionalProperties: true
+          type: object
+        runtime_mappings:
+          additionalProperties: true
+          type: object
+        size:
+          minimum: 0
+          type: integer
+        sort:
+          $ref: '#/components/schemas/Security_Detections_API_AlertsSort'
```

Comment:
I always have trouble finding the source of these references to shared files. Can you point me towards the file with the content that's being referenced?